core/src/main/java/org/apache/carbondata/core/stats/QueryStatisticsConstants.java

```diff
@@ -58,6 +58,14 @@
 
   String PAGE_SCANNED = "The number of page scanned";
 
+  String MEASURE_FILLING_TIME = "measure filling time";
```

Comment:
Can you provide comment, "filling" means after IO read and decoding and copying into the target buffer?